packages/ag-charts-enterprise/src/features/annotations/axisButton.css

```diff
@@ -0,0 +1,31 @@
+.annotations-axis-button {
```

Comment:
Please use bem css naming - https://getbem.com/naming/

And it's best to avoid referencing the element type like `button` directly.

e.g.

```css
.ag-charts-annotations__axis-button-wrapper {}
.ag-charts-annotations__axis-button {}
.ag-charts-annotations__axis-button--hidden {}
```

and https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-community/src/chart/toolbar/toolbarStyles.css